channels/stable

```diff
@@ -32,13 +32,13 @@ spec:
       kubenet: {}
   kubernetesVersions:
   - range: ">=1.10.0"
-    recommendedVersion: 1.10.3
+    recommendedVersion: 1.10.5
```

Comment:
i think we can do 1.10.6 now

Reply:
am i misunderstanding what's going on here? is the point here that we've used these versions in alpha already so they're known quantities?